pkg/controller/statusmanager/status_manager.go

```diff
@@ -21,14 +22,12 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
-type StatusLevel int
```

Comment:
why did you get rid of the `StatusLevel` type?

Reply:
I thought it's more explicit when reading the code and seeing the that real type of `StatusLevel` is an `int`, directly.

Reply:
but it's not an arbitrary `int`. You can't do `SetNotDegraded(42)`.

Doing aliases for `int` like this is golang's equivalent of enums

Reply:
Ahh, I didn't know that!